### PR TITLE
playbook: [portfolio] Moving signup to top

### LINKED ISSUE #247: workshop: Portfolio
Moving signing up to the top to avoid loosing work and confusion with Github and JSBin. @jonleung

### PR DESCRIPTION
**Pull request by saescapa**
@jonl Here is my version, signup to the top.

fixes #247

### CODE REVIEW

**Comment by paked** [comment]
:+1: LGTM

**Comment by MaxWofford** [comment]
@jonleung Any updates on this?

**Comment by zachlatta** [comment]
Any updates on this @jonleung?

**Comment by jonleung** [comment]
Moving this into todo queue. Sorry for the delay @saescapa

**Comment by paked** [comment]
@jonleung you're pretty busy right now. How about @zachlatta or @MaxWofford merge this in for the sake of contribution streamlining?

**Comment by jonleung** [comment]
Thanks a good suggestion @paked!

For now, I think I'd rather validate that I like the flow of this myself. I'd rather take longer to merge it in because the workshop currently works now (albeit not the best but well enough) and I'd opt for not breaking it (I'm super picky lol).

**Comment by MaxWofford** [comment]
@jonleung Perhaps we should close this and opt to reopen it if it is ever needed?

**Comment by jonleung** [comment]
I'd like to leave it open for now.

**Comment by jonleung** [comment]
Because I think it is an important change.

**Comment by jonleung** [comment]
For clarity, I'm planning to merge this as soon as I finish my tasks this week!

File: playbook/workshops/portfolio/README.md [hunk 1]
```diff
@@ -19,12 +19,60 @@ To do this, you will be learning the basics of two languages: HTML and CSS.
 
 Every website that you have ever seen are written in these two languages.
 
+## Creating a JS Bin Account
+
+Before anything, let's create an account in [http://jsbin.com](http://jsbin.com)
+!
```
Comment (zachlatta) [inline playbook/workshops/portfolio/README.md]: Should this exclamation mark be on the next page?

**Comment by jonleung** [comment]
I've made a new pull request #417 to merge your changes in @saescapa! I've added some changes on top of your changes. Closing this pull request in favor of #417.

**Comment by zachlatta** [comment]
I'd personally leave this PR open and let #417 close this PR once it's merged. If #417's description has "Closes #267" in it, it'll automatically close this PR when merged

**Comment by jonleung** [comment]
Opening due to @zachlatta's suggestion.

**Comment by zachlatta** [comment]
Merged in #455. Thank you @saescapa!

**Comment by MaxWofford** [comment]
@zachlatta's gif game going strong tonight

**Pull request closed without merging**